src/components/Api.js

```diff
@@ -1,43 +1,46 @@
-import Section from './Section.js';
-import {createNewCard} from '../utils/utils.js';
-import {cardContainerClass} from '../utils/constants.js';
-import { userInfo, formValidatorCardModal } from '../pages/index.js';
-
 export default class Api{
-    constructor(){
-      this._baseUrl = 'https://around-api.en.tripleten-services.com/v1',
+    constructor(loadUserInfoCallback, loadCardsCallback, editProfileCallback, addNewCardCallback){
+      this._baseUrl = 'https://around-api.en.tripleten-services.com/v1';
```

Comment:
```js

    this._baseUrl = "https://around-api.en.tripleten-services.com/v1";
    this._headers = {
      authorization: "4c45d989-e1aa-4bb6-a467-6ac9c46f3dac",
      "Content-Type": "application/json",
    };
```

This is a hardcode inside `Api`. It should come from `index.js` as parameters of the constructor


```js
  constructor(params) {
    this._baseUrl = params.baseUrl
    this._headers = params.headers
```